test/jobs/create-initial-pr-comment.js

```diff
@@ -85,98 +58,204 @@ test('create-initial-pr-comment', async t => {
-    await repositories.remove(await repositories.get('42'))
-    await repositories.remove(await repositories.get('42:branch:deadbeef'))
-    await repositories.remove(await repositories.get('42:pr:1234'))
+    const { installations, repositories } = await dbs()
+    await Promise.all([
+      removeIfExists(installations, '123'),
+      removeIfExists(repositories, '42', '42:branch:deadbeef', '42:pr:1234')
+    ])
   })
 
-  t.test('create comment for initial pr created by user', async t => {
-    t.plan(2)
+  test('create comment for initial pr created by user', async () => {
+    expect.assertions(2)
 
-    githubNock
+    nock('https://api.github.com')
+      .post('/installations/123/access_tokens')
+      .optionally()
+      .reply(200, {
+        token: 'secret'
+      })
+      .get('/rate_limit')
+      .optionally()
+      .reply(200, {})
+      .get('/repos/finnp/test')
+      .reply(200, {
+        name: 'test',
+        html_url: 'https://github.com/finnp/test',
+        newBranch: 'greenkeeper/initial'
+      })
       .get('/repos/finnp/test/issues/1234')
       .reply(200, {
         state: 'open',
         locked: false
       })
       .post('/repos/finnp/test/issues/1234/comments')
       .reply(201, () => {
-        t.pass('comment added')
+        // comment added
+        expect(true).toBeTruthy()
         return {}
       })
 
-    await runWorker()
+    const { repositories } = await dbs()
+    const branchDoc = await repositories.get('42:branch:deadbeef')
+    const repository = await repositories.get('42')
+
+    await createInitialPrComment({
+      accountId: '123',
+      branchDoc,
+      prDocId: '42:pr:1234',
+      repository,
+      combined: {state: 'success'},
+      installationId: '123'
+    })
 
     const prDoc = await repositories.get('42:pr:1234')
-    t.ok(prDoc.initialPrCommentSent, 'initialPrCommentSent set to true')
+    expect(prDoc.initialPrCommentSent).toBeTruthy()
     await waitFor(timeToWaitAfterTests)
   })
 
-  t.test('does nothing if the repo has already received the comment', async t => {
-    t.plan(1)
+  test('does nothing if the repo has already received the comment', async () => {
+    expect.assertions(1)
 
+    const { repositories } = await dbs()
     await upsert(repositories, '42:pr:1234', {initialPrCommentSent: true})
 
-    githubNock
+    nock('https://api.github.com') // eslint-disable-line
+      .post('/installations/123/access_tokens')
+      .optionally()
+      .reply(200, {
+        token: 'secret'
+      })
+      .get('/rate_limit')
+      .optionally()
+      .reply(200, {})
+      .get('/repos/finnp/test')
+      .reply(200, {
+        name: 'test',
+        html_url: 'https://github.com/finnp/test',
+        newBranch: 'greenkeeper/initial'
+      })
       .get('/repos/finnp/test/issues/1234')
       .reply(200, () => {
-        t.fail('Should not query issue status')
+        // Should not query issue status
+        expect(false).toBeFalsy()
```

Comment:
same here, could be replaced with https://github.com/jest-community/jest-extended `.fail`